case-lib/lib.sh

```diff
@@ -1187,3 +1187,37 @@ perf_analyze()
     fi
 }
 
+# test-mic_privacy.sh needs to control mic privacy settings (on/off)
+# needs usbrelay package: https://github.com/darrylb123/usbrelay
+# param1: switch name
+# param2: switch state
+usbrelay_switch() {
+    switch_name=$1
+    state=$2
+    dlogi "Setting usbrelay switch $switch_name to $state."
+    if ! usbrelay "$switch_name=$state" >/dev/null 2>&1; then
```

Comment:
You can easily avoid negations
```suggestion
    usbrelay "$switch_name=$state" >/dev/null 2>&1 || {
```

Reads as plain english "do or skip"